network/src/commonMain/kotlin/com/wire/kalium/network/api/base/authenticated/conversation/ConversationMemberChangeDTO.kt

```diff
@@ -18,36 +18,14 @@ sealed class ConversationMemberAddedDTO {
     // TODO: the server response with an event aka, UserAdded model is inaccurate
     object Unchanged : ConversationMemberAddedDTO()
 
-    @Serializable
-    @SerialName("conversation.member-join")
-    data class Changed(
-        @SerialName("type")
-        val eventType: String,
-        @SerialName("qualified_conversation")
-        val qualifiedConversationId: ConversationId,
-        @SerialName("qualified_from")
-        val fromUser: UserId,
-        @SerialName("time")
-        val time: String
-    ) : ConversationMemberAddedDTO()
+    data class Changed(val event: EventContentDTO.Conversation.MemberJoinDTO) : ConversationMemberAddedDTO()
 }
 
 @Serializable
 sealed class ConversationMemberRemovedDTO {
     // TODO: the server response with an event aka, UserAdded model is inaccurate
```

Comment:
What about this `TODO`?

Reply:
Good question. I'll find out who added it. I suppose it could be removed.